Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -355,7 +357,7 @@ private void ConfigureCoreDatahubServices(IServiceCollection services)
 
             services.AddScoped<UpdateProjectMonthlyCostService>();
             services.AddScoped<IProjectCreationService, ProjectCreationService>();
-
+            services.AddScoped<IProjectUserManagementService, ProjectUserManagementService>();
             services.AddDatahubApplicationServices();
             services.AddDatahubInfrastructureServices(Configuration);
```

Comment:
Let's put the `AddScoped<IProjectUserManagementService, ProjectUserManagementService>()` inside the `AddDatahubInfrastructureServices(Configuration)` call - working on cleaning up this giant startup file lol

Reply:
Fixed in this commit https://github.com/ssc-sp/datahub-portal/commit/bea6e5abd8a53d237d5acdcd87c0a6e6b0d1d3a4